components/breadcrumbs/breadcrumbs.js

```diff
@@ -0,0 +1,88 @@
+import './breadcrumb.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+/**
+ * A component to help users understand where they are within the LMS. would be used the parent of d2l-breadcrumb and will render them in a slot
+ */
+class BreadCrumbs extends RtlMixin(LitElement) {
+	static get properties() {
+		return {
+			/**
+			 * indicates whether the breadcrumbs are redenred in compact mode (which means it will only display the last time and also includes some visual differences)
+			 */
+			compact: {
+				type: Boolean,
+				reflect: true
+			}
+		};
+	}
+
+	static get styles() {
+		return css`
+			:host {
+				display: block;
+				position: relative;
+			}
+
+			:host([hidden]) {
+				display: none;
+			}
+
+			:host::after {
+				background: linear-gradient(to right, rgba(255, 255, 255, 0), rgb(251, 252, 252));
+				bottom: 0;
+				content: '';
+				pointer-events: none;
+				position: absolute;
+				right: 0;
+				top: 0;
+				width: 10px;
+			}
+
+			:host([dir="rtl"])::after {
+				background: linear-gradient(to left, rgba(255, 255, 255, 0), rgb(251, 252, 252));
+				left: 0;
+				right: auto;
+			}
+
+			.d2l-breadcrumbs-wrapper {
+				overflow: hidden;
+				position: relative;
+				white-space: nowrap;
+			}
+
+			.d2l-breadcrumbs-wrapper ol {
+				margin: 0;
+				padding-left: 0;
+				padding-right: 0;
+				list-style: none;
+			}
+
+			:host([compact]) .d2l-breadcrumbs-wrapper ::slotted(d2l-breadcrumb:not(:last-of-type)) {
+				display:none;
+			}
+		`;
+	}
+
+	constructor() {
+		super();
+		this.compact = false;
+	}
+
+	render() {
+		return html`
+			<nav aria-label="Breadcrumb" class="d2l-breadcrumbs-wrapper" @d2l-breadcrumb-connected="${this._handleBreadCrumbConnected}">
```

Comment:
This label is going to need to be localized... also wondering whether it should be configurable. We could start with just localizing it and later if folks needs to set custom labels we could provide a custom label.

Have a look at some of the other components (e.g. more/less) for examples of using the `LocalizeMixin`. I think we can just copy [the existing translations](https://search.d2l.dev/xref/lms/lp/_lang/LangTerms/definitions/Framework.xml?r=174eecc7#272) from the MVC version of this component.

Reply:
@dlockhart I think the existing translations are for singular mode but the breadcrumbs is plural. Is there anyone I can contact to fix that for me? (or use google translate?).

Reply:
also on another note: Do we prefer to use the `LocalizeCoreElement` instead of `LocalizeMixin` and centralize translations or do we like it to be inside the component itself? asking since it seems that we have used `LocalizeCoreElement` everywhere instead of `LocalizeMixin`

Reply:
We've been centralizing them in the `lang/` files and using `LocalizeCoreElement`.

Reply:
The existing translations should work -- they look and behave essentially the same as this. Despite sometimes pluralizing it as "breadcrumbs" you could also can it a "breadcrumb" in reference to the collection, i.e. "breadcrumb trail".

Reply:
I think you could remove the colon from the existing translations though if that's easy.

Reply:
yeah it should be easy. Also there is some minor differences between the langterms from what we have. e.g. there is a `de-de` langterm in LMS which I think would be `de` here. how should I organize that part?

Reply:
Yup, you would just use the `de-de` term in `de`. The mappings are here https://github.com/BrightspaceUI/core/blob/master/core.serge.json for which language-locale combo ends up in each file. That should cover most cases.

Reply:
Done ✔️